Makefile

```diff
@@ -154,6 +154,7 @@ $(PGOROOT)/$(DFSET)/Dockerfile.%.$(DFSET):
 		--build-arg PGVERSION=$(PGO_PG_VERSION) \
 		--build-arg BACKREST_VERSION=$(PGO_BACKREST_VERSION) \
 		--build-arg ANSIBLE_VERSION=$(ANSIBLE_VERSION) \
+		--build-arg PGO_CONF_DIR=$(PGO_CONF_DIR) \
```

Comment:
needs to be pulled out - I was going to try to add a variable for the config directory but that didn't work out